src/components/FAQ/index.jsx

```diff
@@ -0,0 +1,96 @@
+import { Disclosure, Transition } from '@headlessui/react';
```

Comment:
Perhaps changing the file name to 'FAQ.jsx'

Reply:
@sarahaboudalal lets keep those minimal things to the bug bashing part.